src/App.js

```diff
@@ -44,6 +44,7 @@ class App extends Component {
     }
 
     runSql() {
+		console.log("is valid json?",this.validateJSON(this.state.json));
```

Comment:
Actually we should update this on screen, as soon we might see results on screen directly.

Reply:
Yeah maybe we should have this do some type of automatic validation, so the textbox is red and if the user hit's submit it will display an error to that user